src/code.cloudfoundry.org/lib/multiple-cidr-network/multiple_cidr_network.go

```diff
@@ -0,0 +1,57 @@
+package multiple_cidr_network
+
+import (
+	"net"
+)
+
+type MultipleCIDRNetwork struct {
+	Networks     []*net.IPNet
+	SmallestMask int
+}
+
+func NewMultipleCIDRNetwork(cidrs []string) (MultipleCIDRNetwork, error) {
+	var networks []*net.IPNet
+	smallestMask := 0
+	for i, c := range cidrs {
+		_, n, err := net.ParseCIDR(c)
+		if err != nil {
+			return MultipleCIDRNetwork{}, err
```

Comment:
is it worth it to generate all errors for all blocks, and return the sum of all errors, rather than fixing them one at a time?

Reply:
eh, I think I want to keep this one simple.